[FIX] web_widget_datepicker_options: Fix error when field is undefined

luisg123v opened the pull request:
Currently, when a view is rendered and a date or datetime field is not
defined (e.g. when designing reports), it causes a JS error.

This patch fixes the above error.

pedrobaeza commented:
Please change module version number according https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers

luisg123v committed:
[FIX] web_widget_datepicker_options: Fix error when field is undefined

Currently, when a view is rendered and a date or datetime field is not
defined (e.g. when designing reports), it causes a JS error.

This patch fixes the above error.

luisg123v commented:
@pedrobaeza 
Thanks for pointing it out, done.
